deploy/kubernetes/images/stable-master/image.yaml

```diff
@@ -43,6 +43,16 @@ imageTag:
   newTag: "v2.3.0"
 ---
 
+apiVersion: builtin
```

Comment:
I have no idea should this be added to different files as well in `images/` folder?